network/hnswrapper/hnsv2wrapperfake.go

```diff
@@ -400,11 +467,40 @@ func NewFakeHostComputeEndpoint(endpoint *hcn.HostComputeEndpoint) *FakeHostComp
 	}
 }
 
+func (fEndpoint *FakeHostComputeEndpoint) PrettyString() string {
```

Comment:
I realize that existing methods do this, but the generally accepted style in the wider Go community is for a single letter matching the first letter of the type the method is being implemented on for "this":

```go
func (f *FakeHostComputeEndpoint) PrettyString() string
```

Reply:
would you suggest keeping the same style throughout the file or adding new methods in the generally accepted way?

Reply:
linters complain if all method receivers are not named the same, so all should be changed.

Reply:
for the reviewer's sake, can address this in a followup PR since it will create a diff for most of the file